docs/development/decision-records/2024-10-07-azure-distribution/README.md

```diff
@@ -0,0 +1,22 @@
+# Refactor of the Release Workflow
+
+## Decision
+
+We will stop to provide an azure flavoured distribution of the connector.
+
+## Rationale
+
+Two reasons are pushing toward this decision:
+- Tractus-X Connector is the reference implementation for Catena-X: this project is not meant to satisfy different needs
+  but to show how an EDC Connector distribution for Catena-X can be assembled. Our focus should point toward maintaining
+  a single distribution.
+- Tractus-X is an open source software, and we should focus on a distribution that involves OSS services.
```

Comment:
I would also add two more reasons: 
- for the past year there were no resources to stand up an integration test environment, i.e. an actual Azure Vault instance
- there is no reason for favoritism toward Azure Vault over other services like AWS SecretsManager or similar